### PR TITLE
Solved the simple calculator challenge

### PR DESCRIPTION
**Pull request by svetlivaksevski**

### CODE REVIEW

File: js-variables-and-numbers_simple-calculator/js/index.js [hunk 1]
```diff
@@ -61,24 +86,37 @@ const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
 
 increaseByOneButton.addEventListener("click", () => {
   // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+let operantIncrease = operand1 + 1;
```
Comment (Hainebach) [inline js-variables-and-numbers_simple-calculator/js/index.js:89]: you can also make it shorter with operantIncrease++
it automatically adds one

File: js-variables-and-numbers_simple-calculator/js/index.js [hunk 2]
```diff
@@ -61,24 +86,37 @@ const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
...
 
 increaseByFiveButton.addEventListener("click", () => {
   // 8. Increase the value of "operand1" by 5. Log the new value to the console.
+  let operantIncreaseByFive = operand1 + 5;
+  console.log(operantIncreaseByFive);
 });
 
 decreaseByOneButton.addEventListener("click", () => {
   // 7. Decrease the value of "operand1" by 1. Log the new value to the console.
+  let operantDecrease = operand1 - 1;
```
Comment (Hainebach) [inline js-variables-and-numbers_simple-calculator/js/index.js:102]: same like adding one you can decrease one by using the varName--;

**Review by Hainebach — COMMENTED** [review]
it looks good, I would take out the variables out of the functions so you can call them outside of the functions as well.

File: js-variables-and-numbers_simple-calculator/js/index.js (hunk 1 above)
Comment (svetlivaksevski) [reply js-variables-and-numbers_simple-calculator/js/index.js:89]: Thank you

**Pull request merged**